src/SignalR/clients/ts/signalr/tests/MessageSize.test.ts

```diff
@@ -0,0 +1,168 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+import { HubConnection } from "../src/HubConnection";
+import { IConnection } from "../src/IConnection";
+import { IHubProtocol } from "../src/IHubProtocol";
+import { ILogger } from "../src/ILogger";
+import { JsonHubProtocol } from "../src/JsonHubProtocol";
+import { NullLogger } from "../src/Loggers";
+import { Subject } from "../src/Subject";
+import { VerifyLogger } from "./Common";
+import { TestConnection } from "./TestConnection";
+import { delayUntil, registerUnhandledRejectionHandler } from "./Utils";
+
+registerUnhandledRejectionHandler();
+
+function createHubConnection(connection: IConnection, logger?: ILogger | null, protocol?: IHubProtocol | null) {
+    return HubConnection.create(connection, logger || NullLogger.instance, protocol || new JsonHubProtocol());
```

Comment:
Are the TS tests for MessagePack? It looks like these are only testing JsonHubProtocol.

Reply:
No tests for MessagePack, that's what we were talking about Friday when we decided to just get the PR in.